metadata.rb

```diff
@@ -22,7 +22,7 @@
 
 depends 'build-essential', '>= 6.0.0'
 depends 'chef-sugar', '>= 3.2.0'
-depends 'chef-ingredient', '>= 0.18.0'
+depends 'chef-ingredient', '>= 0.21.4'
```

Comment:
This change requires changes from https://github.com/chef-cookbooks/chef-ingredient/pull/128 that were released in `chef-ingredient 0.21.4`.